test/unit/presubmit-custom-runners-tests.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Copyright 2018 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source $(dirname $0)/presubmit-integration-tests-common.sh
+
+function build_tests() {
+  RAN_BUILD_TESTS=1
```

Comment:
should it be `RUN` instead of `RAN`?

Reply:
oh never mind, I got it now

Reply:
just an idea, how about `BUILD_TESTS_RAN` ?

Reply:
I think this is an unnecessary renaming.